parsers/verilog.c

```diff
@@ -1476,6 +1485,57 @@ static int pushEnumNames (tokenInfo* token, int c)
 	return c;
 }
 
+// create a list of struct/union members
+static int pushMembers (tokenInfo* token, int c)
+{
+	if (c == '{')
+	{
+		c = skipWhite (vGetc ());
+		do
+		{
+			verilogKind kind = K_UNDEFINED;
```

Comment:
Is this "kind" used?
The address for this local var is passed to processType().
processType() may fill the variable with a value.
However, the value for the local variable is not used.
If the above my analysis is correct, could you take one of the following action:
* rename the local variable to "dummy_kind", "tmp_kind", "kind_no_use" or something, or
* eliminate the local variable after making processType accept NULL as the third argument.

Reply:
> Is this "kind" used?

Yes.

```
// input
//   kind: kind of context
// output
//   kind: kind of type
//   token: identifier token (unless K_IDENTIFIER nor K_UNDEFINED)
static int processType (tokenInfo* token, int c, verilogKind* kind)
{
```

Reply:
Oh, I see. The value for kind is consumed in processType:
```
		if (c == '{')	// skip enum, struct, or union member
		{
			if (*kind == K_ENUM)
				c = pushEnumNames (token, c);
			else
				c = skipWhite (skipPastMatch ("{}"));
		}

```

Reply:
Reducing the local variable contributes readability of the code.
You can eliminate "kind" with the following technique.  But the technique makes the code just unreadable.

```
void
foo (verilogKind *i)
{
}

int
main(void)
{
  foo (&((struct { verilogKind i; }){ .i = 1 }.i));
}
```

Reply:
I've added a comment.

```
			verilogKind kind = K_UNDEFINED;	// set kind of context for processType()
```

> So ignore this comment.

Of course, I do :-)